cyme/core/simd_vector/simd_vec.hpp

```diff
@@ -29,8 +29,16 @@
 
 #include "cyme/core/simd_vector/trait.hpp"
 #include "cyme/core/simd_vector/simd_wrapper.hpp"
-
+#include <iostream>
 namespace cyme{
+    /**
+    Helper for print function
+    */
+    template<class T>
+    struct elems_helper{
+	static const int size = (cyme::trait_register<T,cyme::__GETSIMD__()>::size)/sizeof(T);
```

Comment:
good but you forget the int N template parameter for the unroll factor so bug, you add this parameter N and you multiply all your computation by N